Formula/pnetcdf.rb

```diff
@@ -0,0 +1,60 @@
+class Pnetcdf < Formula
+  desc "Parallel netCDF library for scientific data using the OpenMPI library"
+  homepage "https://parallel-netcdf.github.io/index.html"
+  url "https://parallel-netcdf.github.io/Release/pnetcdf-1.11.2.tar.gz"
+  sha256 "d2c18601b364c35b5acb0a0b46cd6e14cae456e0eb854e5c789cf65f3cd6a2a7"
+
+  depends_on "gcc"
```

Comment:
Is this a runtime dependency as well?

Reply:
The resulting library is linked against `libgfortran.5.dylib`, which comes from the gcc package.
gcc is already implicitly a dependency through open-mpi. What is the recommended way of expressing the dependencies here?

Reply:
I guess it's fine then. It's just rare for gcc to be required runtime.